goaviatrix/security_domain.go

```diff
@@ -67,6 +67,21 @@ type IntraDomainInspection struct {
 	IntraDomainInspectionEnabled bool   `json:"intra_domain_inspection"`
 }
 
+type DomainDetails struct {
```

Comment:
Maybe it's better to be called `NetworkDomainDetails`? I'm not sure if we will have other kinds of domains that might get confused.

Reply:
yes, i have fixed as NetworkDomainDetails